Formula/doxygen.rb

```diff
@@ -1,9 +1,9 @@
 class Doxygen < Formula
   desc "Generate documentation for several programming languages"
   homepage "https://www.doxygen.nl/"
-  url "https://doxygen.nl/files/doxygen-1.9.7.src.tar.gz"
+  url "https://doxygen.nl/files/doxygen-1.9.7.repack.src.tar.gz"
```

Comment:
It's detecting the version as `1.9.7.repack.src` so needs a:

```suggestion
  url "https://doxygen.nl/files/doxygen-1.9.7.repack.src.tar.gz"
  version "1.9.7"
```